tests/unit_tests.py

```diff
@@ -179,6 +179,17 @@ def test_download_html(self):
         self.assertEqual(75406, len(self.article.html))
 
     @print_test
+    def test_meta_refresh_redirect(self):
+        config = Configuration()
+        config.follow_meta_refresh = True
+        article = Article(
+            '', config=config)
+        html = mock_resource_with('google_meta_refresh', 'html')
+        article.download(html=html)
+        article.parse()
+        self.assertEqual(article.title, 'Example Domain')
```

Comment:
Does this actually pull from example.com? I'm not sure it's such a good idea to do that during tests, but can't think of other alternatives that will fit into the current mocking pattern...